src/utils/webviewUtils.ts

```diff
@@ -0,0 +1,54 @@
+import { leetCodeExecutor } from "../leetCodeExecutor";
+import { Command, IProblem } from "../shared";
+
+export async function renderHTML(node: IProblem): Promise<string> {
+    const description: string = await leetCodeExecutor.getDescription(node);
+    const descriptionHTML: string = description.replace(/\n/g, "<br>");
+    const htmlTemplate: string = `
+    <!DOCTYPE html>
+    <html>
+        <head>
+            <meta charset="UTF-8">
+            <meta name="viewport" content="width=device-width, initial-scale=1.0">
+            <title>Preview Problem</title>
+        </head>
+        <style>
+            #solve {
+                position: fixed;
+                bottom: 10px;
+                right: 10px;
+                border: 0;
+                margin: 10px 0;
+                padding: 2px 14px;
+                color: white;
+                background-color: rgb(14, 99, 156);
```

Comment:
Hard code the color is not recommended because the appearance will not be consistent when the user switches the theme. 

Instead, you can use the VS Code embed vars, for example `background-color: var(--vscode-editor-background);`

If you want to see all the available variables, open a web view, trigger the command: `Developer: Open Webview Developer Tools`, inspect an element, then you can see all the available variables.

![image](https://user-images.githubusercontent.com/6193897/53941213-7cb4d580-40f2-11e9-858a-5e255bbeeb25.png)